documentation/modules/auxiliary/scanner/http/citrix_directory_traversal.md

```diff
@@ -0,0 +1,51 @@
+##  Introduction
+
+An issue was discovered in Citrix Application Delivery Controller (ADC) and Gateway 10.5, 11.1, 12.0, 12.1, and 13.0. The vulnerability, tracked as CVE-2019-19781, allows for directory traversal. If exploited, it could allow an unauthenticated attacker to perform arbitrary code execution.
+
+Because vulnerable servers allow for directory traversal, they will accept the request `GET /vpn/../vpns/` and  process it as a request for `GET /vpns/`, a directory that contains PERL scripts that can be targeted to allow for limited file writing on the vulnerable host.
+
+This module checks if a target server is vulnerable by issuing an HTTP GET request for `/vpn/../vpns/cfg/smb.conf`and then checking the response for `global`since this configuration file should contain global variables. If ``global``is found, the server is vulnerable to CVE-2019-19781.
+
+## Verification Steps
+
+1. Install the module as usual
+
+2. Start msfconsole
+
+3. Do: `use auxiliary/scanner/http/citrix_directory_traversal`
+
+4.  Do: `set RHOSTS [IP]`
+
+5. Do: `run`
+
+
+## Options
+
+1.  `Proxies`  . This option is not set by default.
+2.  `RPORT`  . The default setting is  `80`. To use:  `set RPORT [PORT]`
+3.  `SSL`  . The default setting is  `false`.
+4.  `THREADS`  . The default setting is  `1`.
+5.  `VHOST`  . This option is not set by default.
```

Comment:
This formatting is inconsistent with existing documentation. See examples:

* [Template](https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/module_doc_template.md)
* [Examples](https://github.com/rapid7/metasploit-framework/tree/master/documentation/modules)